r/README.Rmd

```diff
@@ -16,48 +18,77 @@ knitr::opts_chunk$set(
 
 [![conda-forge](https://img.shields.io/conda/vn/conda-forge/r-arrow.svg)](https://anaconda.org/conda-forge/r-arrow)
 
-R integration with Apache Arrow.
+[Apache Arrow](https://arrow.apache.org/) is a cross-language development platform for in-memory data. It specifies a standardized language-independent columnar memory format for flat and hierarchical data, organized for efficient analytic operations on modern hardware. It also provides computational libraries and zero-copy streaming messaging and interprocess communication.
+
+The `arrow` package exposes an interface to the Arrow C++ library.
 
 ## Installation
 
-You first need to install the C++ library:
+Installing `arrow` is a two-step process: first install the Arrow C++ library, then the R package.
 
-### macOS
+### Release version
 
-On macOS, you may install using homebrew:
+The current release of Apache Arrow is 0.13.
```

Comment:
We need to update the current release version automatically on our release process when we want to embed the current release version.
https://github.com/apache/arrow/blob/master/dev/release/00-prepare.sh#L96-L110
https://github.com/apache/arrow/blob/master/dev/release/00-prepare-test.rb#L126-L139

Reply:
Created https://issues.apache.org/jira/browse/ARROW-5415 for this.